blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/nav-widget.jsx

```diff
@@ -36,20 +36,24 @@ const NavWidget = ({
 				iconRight={<Icon name="HamburgerMenu" />}
 				variant="secondary-reverse"
 				size="small"
+				data-testid="nav-chain-nav-section-button"
 			>
 				{breakpoint === "desktop" ? phrases.t("header-nav-chain-block.sections-button") : null}
 			</Button>
 		) : null);
 
+	// The children prop is not 100% guaranteed to be an array, so this is a bit of future proofing in case something changes.
```

Comment:
appreciate the comment and effort, but can we remove it?  (the comment, not the code)